auth/exported-token.service.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function ExportedTokenService (options) {
+  this.initialize = function () {
+    return Promise.resolve(this)
+  }
+  this.getConsumer = function (req) {
+    const id = req.get('X-User-Id')
+    const name = req.get('X-User-Name')
+    let scopes = req.get('X-User-Scopes')
+    if (scopes) scopes = scopes.split(' ')
```

Comment:
the same variable name is used for both an array and a string. This seems quite confusing to me. Please use 2 variables names & const.